libfs/journal_control_file.go

```diff
@@ -55,6 +63,12 @@ func (a JournalAction) Execute(
 			return err
 		}
 
+	case JournalPause:
+		jServer.PauseAutoFlush(ctx, tlf)
```

Comment:
Yeap, but that's not intended: see https://keybase.atlassian.net/browse/KBFS-1433